src/libraries/System.Transactions.Local/tests/OleTxTests.cs

```diff
@@ -496,6 +496,12 @@ public void GetDtcTransaction()
     [ConditionalFact(nameof(IsRemoteExecutorSupportedAndNotNano))]
     public void Distributed_transactions_require_ImplicitDistributedTransactions_true()
     {
+        // Temporarily skip on 32-bit where we have an issue.
+        if (!Environment.Is64BitProcess)
```

Comment:
Can't that be blocked on 32-bit platforms by modifying the `ConditionalFact`? I think I've seen this on other tests.

Reply:
I admit that between IsRemoteExecutorSupported and the other "skip" stuff in this test suite, I just went for the easy thing here. Yeah, it could probably be cleaned up a bit, but it's probably not super important either.